src/alembic/versions/006_8638e949eea1_alter_customer_postal_code_to_varchar.py

```diff
@@ -0,0 +1,46 @@
+"""alter customer_postal_code to varchar
+
+Revision ID: 8638e949eea1
+Revises: 775b2cf7ab94
+Create Date: 2024-09-16 15:12:30.287594
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "8638e949eea1"
+down_revision: Union[str, None] = "775b2cf7ab94"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.alter_column("sale_transaction", "customer_postal_code", type_=sa.String())
+    op.alter_column("use_transaction_location", "customer_postal_code", type_=sa.String())
+    op.alter_column("use_transaction_longitudinal", "customer_postal_code", type_=sa.String())
+
+
+def downgrade() -> None:
+    op.alter_column(
+        "sale_transaction",
+        "customer_postal_code",
+        type_=sa.BigInteger(),
+        postgresql_using="customer_postal_code::bigint",
+    )
+    op.alter_column(
+        "use_transaction_location",
+        "customer_postal_code",
+        type_=sa.BigInteger(),
+        postgresql_using="customer_postal_code::bigint",
+    )
+    op.alter_column(
+        "use_transaction_longitudinal",
+        "customer_postal_code",
+        type_=sa.BigInteger(),
+        postgresql_using="customer_postal_code::bigint",
+    )
```

Comment:
I'm not sure if it's worth including these downgrade commands. If the columns have non-integer values in them, the downgrade will fail. 

Not sure if it would be better to omit the downgrade or run it as a drop and add new column operation, which would lose all the data in the existing column, but not fail.

Reply:
I know. I'm just trying to have something. I'd hate to lose data.